libs/coin-modules/coin-stellar/src/logic.ts

```diff
@@ -0,0 +1,371 @@
+import type { CacheRes } from "@ledgerhq/live-network/cache";
+import { makeLRUCache } from "@ledgerhq/live-network/cache";
```

Comment:
It could be nice to define and use all of this cache methods, within network calls.